src/artm_tests/regularizers_test.cc

```diff
@@ -305,8 +305,8 @@ TEST(Regularizers, NetPlsa) {
 
 // artm_tests.exe --gtest_filter=Regularizers.RelativeRegularization
 TEST(Regularizers, RelativeRegularization) {
-  int nTopics = 500;
-  int nTokens = 500;
+  int nTopics = 50;
```

Comment:
did the test became too slow? (just need to convince myself that introduction of transactions doesn't change results of existing tests)